### ISSUE TITLE
Check validity of DC file before sourcing

### ISSUE THREAD

**Issue by mschnitzer** [openSUSE/daps#269]
Let's say I want to execute this command: 
`daps -d LICENSE list-srcfiles --nodc --noent --noimg`

Because the license file is not a valid DC file the console output this error:

```
/home/manuel/GitHub/docmanager/LICENSE: line 1: GNU: command not found
/home/manuel/GitHub/docmanager/LICENSE: line 2: Version: command not found
/home/manuel/GitHub/docmanager/LICENSE: line 4: syntax error near unexpected token `C'
/home/manuel/GitHub/docmanager/LICENSE: line 4: ` Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>'
basename: missing operand
Try 'basename --help' for more information.
ERROR: XML files must reside in an "xml/" directory. "/home/manuel/GitHub/docmanager/xml/" is not a valid location.
```

Anyway, sourcing is a security risk. Maybe we should write a parser for parsing DC files.

**Comment by fsundermeyer** [comment]
```
bash -n <DCFILE>
```

**Comment by ghost** [comment]
Since the DAPS source may soon include an INI parser ... maybe DC files could give way to INIs..?

**Comment by fsundermeyer** [comment]
Has been fixed with 8bca9df3cc (config file parser for DAPS)
